src/common/options/rgw.yaml.in

```diff
@@ -3237,13 +3237,53 @@ options:
   type: str
   level: advanced
   desc: experimental Option to set backend store type
-  long_desc: defaults to rados. Other valid values are dbstore(experimental).
+  long_desc: defaults to rados. Other valid values are dbstore and motr(experimental).
   default: rados
   services:
   - rgw
   enum_values:
   - rados
   - dbstore
+  - motr
+- name: motr_profile_fid
+  type: str
+  level: advanced
+  desc: experimental Option to set Motr profile fid
+  long_desc: example value 0x7000000000000001:0x4f
+  default: 0x7000000000000001:0x0
+  services:
+  - rgw
+- name: my_motr_fid
```

Comment:
Can we rename these motr_my_fd and motr_my_endpoint (or something), so that all motr related config options are prefixed with motr_?

Reply:
Yes, let me fix that...

Reply:
Done.